TheWarWithin/Priorities/MageFrost.simc

```diff
@@ -29,8 +29,8 @@ actions.aoe+=/cone_of_cold,if=target.within10&!talent.coldest_snap&active_enemie
 actions.aoe+=/frostbolt
 actions.aoe+=/call_action_list,name=movement
 
-actions.cds+=/use_item,name=imperfect_ascendancy_serum,if=buff.icy_veins.remains>19|boss&fight_remains<25
-actions.cds+=/use_item,name=spymasters_web,if=(buff.icy_veins.remains>19&(boss&fight_remains<100|buff.spymasters_report.stack=40&fight_remains>120))|boss&fight_remains<25
```

Comment:
Is this an intentional change?  We probably want/need to check stacks before using the trinket.

Reply:
not sure what happened there, I would add them back in.